panoptes_aggregation/tests/router_tests/test_routes.py

```diff
@@ -68,6 +68,29 @@ def test_one_running_reducer_route(self):
                     running_reducer_name
                 )
 
+    # Override json.dumps() for this test so it doesn't try to jsonify the mock
+    import json
+    @patch("json.dumps", return_value=json.dumps({'project_id': 1, 'workflow_id': 10, 'user_id': 100, 'task_id': 'asdf'}))
```

Comment:
I think you want to patch the `json` import from `panoptes_aggregation.batch_aggregation` (although it likely does the same thing in this case, changing it will scop the patch to only the `json.dump` in that file rather than globally across the app).
```
@patch("panoptes_aggregation.batch_aggregation.json.dumps", return_value=json.dumps({'project_id': 1, 'workflow_id': 10, 'user_id': 100, 'task_id': 'asdf'}))
```

Reply:
✔️